Python/Product/Debugger.Concord/DebuggerOptions.cs

```diff
@@ -20,6 +20,9 @@
 namespace Microsoft.PythonTools.Debugger.Concord {
     public static class DebuggerOptions {
         // These are intentionally not implemented as auto-properties to enable easily changing them at runtime, including when stopped in native code.
+        //
+        // These show up in commands in the debugger watch window if you set the registry value:
+        // HKCU/Software/Microsoft/PythonTools/Debugger PythonDeveloper: DWORD = 1
```

Comment:
Is the space between Debugger and PythonDebugger intentional? Not supposed to be a slash?

Reply:
Yes it's supposed to identify the key and the value. I could be more explicit.